### PR TITLE
Testing indicators

### PR DESCRIPTION
**Pull request by denyskulykov**

### CODE REVIEW

File: tests/functional/tests/test_indicator.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+from ctrlibrary.core.utils import get_observables
+from ctrlibrary.threatresponse.enrich import enrich_observe_observables
+
+
+def test_positive_indicator(module_headers):
+    """Perform testing for enrich observe observables endpoint to get
+    indicators for observable from Pulsedive
+
+    ID: CCTRI-
```
Comment (mistergreen22) [inline tests/functional/tests/test_indicator.py:9]: you forgot store id

**Comment by mistergreen22** [comment]
maybe in this case too you can check also "domain" and "url" type.

**Comment by oshtaier** [comment]
Implementation using `if` looks really bad in that scenario. Think about key-value pairs or switch/case as last resort

File: tests/functional/tests/test_indicator.py [hunk 2]
```diff
@@ -36,18 +38,53 @@ def test_positive_indicator(module_headers):
...
+
+
+@pytest.mark.parametrize(
+    'observable,observable_type',
+    (('185.141.25.242', 'ip'),
+     ('yizaiwl.cc', 'domain'),
+     ('https://www.google.com/', 'url'))
+)
+def test_positive_indicators(module_headers, observable, observable_type):
+    """Perform testing for enrich observe observables endpoint to get
```
Comment (oshtaier) [inline tests/functional/tests/test_indicator.py:60]: you have same docstring here, so it is not obvious why we have two tests. Please correct that

File: tests/functional/tests/test_indicator.py [hunk 3]
```diff
@@ -36,18 +38,53 @@ def test_positive_indicator(module_headers):
...
+    indicators for observable from Pulsedive
+
+    ID: CCTRI-910-db541ad9-22a2-469b-bb23-cceaab72b56d
+
+    Steps:
+        1. Send request to enrich deliberate observable endpoint
```
Comment (oshtaier) [inline tests/functional/tests/test_indicator.py:66]: `enrich observe observable`

File: tests/functional/tests/test_indicator.py [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
-def test_positive_indicator(module_headers):
+def test_positive_indicator_base(module_headers):
     """Perform testing for enrich observe observables endpoint to get
     indicators for observable from Pulsedive
 
-    ID: CCTRI-
+    ID: CCTRI-910-22c91fdc-31e7-4951-993e-c2a90a92d434
 
     Steps:
         1. Send request to enrich deliberate observable endpoint
```
Comment (oshtaier) [inline tests/functional/tests/test_indicator.py:14]: `enrich observe observable`

File: tests/functional/tests/test_indicator.py (hunk 2 above)
Comment (oshtaier) [reply tests/functional/tests/test_indicator.py:60]: something like `test_positive_indicators_by_type`

File: tests/functional/tests/test_indicator.py [hunk 5]
```diff
@@ -1,12 +1,14 @@
+import pytest
+
 from ctrlibrary.core.utils import get_observables
 from ctrlibrary.threatresponse.enrich import enrich_observe_observables
 
 
-def test_positive_indicator(module_headers):
+def test_positive_indicator_base(module_headers):
```
Comment (oshtaier) [inline tests/functional/tests/test_indicator.py:7]: and here maybe `test_positive_indicator_details`

**Pull request merged**